src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS
+ * properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((result, style) => {
+    const stylePair = style.split(':');
+    const key = stylePair[0] ? stylePair[0].trim() : undefined;
+    const value = stylePair[1] ? stylePair[1].trim() : undefined;
```

Comment:
If you do just `const key = stylePair[0]?.trim();` it will work, if you want to specify nothing it is better to use `null` in such cases, because `undefined` it is variable that has been declared but does not have value